packages/common/editors/EditorContainer.js

```diff
@@ -51,11 +54,27 @@ class EditorContainer extends React.Component {
   }
 
   componentWillUnmount() {
+    document.removeEventListener('click', this.handleDocumentClick);
     if (!this.changeCommitted && !this.changeCanceled) {
       this.commit({ key: 'Enter' });
     }
   }
 
+  handleDocumentClick = (e) => {
+    const { target } = e;
+    const { container, editor } = this;
+    const isClickedInsideEditorContainer = container !== null && (target === container || container.contains(target));
+    let isClickedInsideEditor = false;
+    if (editor !== null) {
```

Comment:
As above...

```suggestion
    if (editor) {
```
```suggestion
    if (!!editor) {
```

Reply:
> React will assign the current property with the DOM element when the component mounts, and assign it back to null when it unmounts

I usually prefer an explicit check than using the `!!` operator but either works